### PR TITLE
update to libgee-0.20.1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: components/library/libgee/Makefile [hunk 1]
```diff
@@ -19,33 +19,40 @@
...
+COMPONENT_SUMMARY=  GObject based collection and utility library
+COMPONENT_LICENSE = LGPLv2.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+COMPONENT_PROJECT_URL=  https://wiki.gnome.org/Projects/Libgee
+COMPONENT_SRC=      $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=  $(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+    sha256:bb2802d29a518e8c6d2992884691f06ccfcc25792a5686178575c7111fea4630
+COMPONENT_ARCHIVE_URL=  https://download.gnome.org/sources/libgee/0.20/$(COMPONENT_ARCHIVE)
+COMPONENT_BUGDB=    library/libgee
```
Comment (alarcher) [inline components/library/libgee/Makefile]: COMPONENT_BUGDB can be removed.

File: components/library/libgee/Makefile [hunk 2]
```diff
@@ -19,33 +19,41 @@
...
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
-COMPONENT_ARCHIVE_HASH=	\
-    sha256:cf9188b225a4e4b84696a13e5f9af0af0aa19a9ed05d01d1161023069f5be3d6
-COMPONENT_ARCHIVE_URL=	http://download.gnome.org/sources/libgee/0.8/$(COMPONENT_ARCHIVE)
-COMPONENT_BUGDB=	library/libgee
+COMPONENT_NAME=     libgee
+COMPONENT_VERSION=  0.20.1
+COMPONENT_SUMMARY=  GObject based collection and utility library
+COMPONENT_LICENSE = LGPLv2.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
```
Comment (alarcher) [inline components/library/libgee/Makefile]: COMPONENT_LICENSE_FILE can be removed.

File: components/library/libgee/Makefile [hunk 3]
```diff
@@ -19,33 +19,41 @@
...
 
-CONFIGURE_OPTIONS  +=		--disable-static
-CONFIGURE_OPTIONS  +=		--enable-shared
+CONFIGURE_OPTIONS  +=       --disable-static
+CONFIGURE_OPTIONS  +=       --enable-shared
 
-# common targets
-build:		$(BUILD_32_and_64)
+# gobject-introspection
+COMPONENT_BUILD_ENV += CC="$(CC)"
```
Comment (alarcher) [inline components/library/libgee/Makefile]: Does gobject-introspection pick the right compiler?

**Review by alarcher — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
gobject-introspection removed - OK

COMPONENT_BUGDB removed

COMPONENT_LICENSE_FILE removed in Makefile

Does the actual file libgee.license need to be removed also?

**Comment by alarcher** [comment]
No no it's just that macros have default values so they do not need to be explicitely defined in the Makefile if they take the default value.

**Review by alarcher — APPROVED** [review]

**Comment by alarcher** [comment]
Gahh.. click on the wrong button... sorry guys... please @agrellum can you squash the changes next time so I do not fail miserably with the merge... :S

**Pull request merged**